Games/Warcraft III TFT/Online/script.js

```diff
@@ -0,0 +1,13 @@
+include(["Functions", "QuickScript", "OnlineInstallerScript"]);
+include(["Functions", "Verbs", "corefonts"]);
```

Comment:
`corefonts` and `vcrun2015` is not used in this script, please remove the includes.